lib/log_parser.rb

```diff
@@ -54,4 +55,30 @@ def parse_kill_line(line)
   def parse_info_change_line(line)
     line.split('n\\')[1].split('\\')[0]
   end
+
+  def count_kills
+    kills = Hash.new(0)
```

Comment:
```suggestion
    kills = {}
```

Só declarando a hash aqui não daria certo?

Reply:
Não, se eu só declarar assim, ao inserir um novo elemento ele vem como `nil` no lugar de 0, e na hora de incrementar acontece um erro